CODE_OF_CONDUCT.md

```diff
@@ -18,6 +18,7 @@ Examples of unacceptable behavior by participants include:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or advances
 * Trolling, insulting/derogatory comments, and personal or political attacks
+* Posting links to insulting, derogatory, or attacking content (except when discussing moderation of content or links already posted in the community)
```

Comment:
This point is more nuanced than the others on the list; there are a lot of contextual factors that would affect whether I see a particular link to such content as acceptable or unacceptable. I don't think we should have items on the list of unacceptable behavior that aren't clear-cut prohibitions. This isn't meant to be exhaustive, or a legalistic procedure for deciding what is or is not acceptable behavior.

Reply:
Maybe I should try to word it in the scope part about directly linked content is treated close to quoted. It is not from the point of view of legalistic procedure, more like documentation.

The code is more useful the better it communicates the policies actually followed, and would prefer to take Jonathan's question «so which of the parts» at face value (and make it clear from the text as posted).

Reply:
> directly linked content is treated close to quoted

I don't think that's quite it either. The link is the expression in our space; like any other expression, its interpretation is down to context. In many contexts, a posted link will be seen as an endorsement of the linked subject, in which case the linker's expression of endorsement is what gets judged against our community values. This interpretation is particularly likely if the linker is also the author of the linked content. But it's also easily possible to link content that doesn't comport with our values in a context that gets interpreted as the opposite of an endorsement, and that can be fine. One could also link perfectly innocuous content in a negative context, and once again it's the interpretation of the expression that matters.

I can't think of a succinct way to summarize this that isn't near-tautological or doesn't paint with too broad a brush. ‘Unacceptable behavior expressed through non-textual media, such as images, links, or emoji, is still unacceptable’?

Reply:
That does sound to me like something rather clearly covering the case in question.

Reply:
I still think `insulting, derogatory, or attacking content`  is still pretty open to interpretation. 

My alternative wording: https://github.com/NixOS/.github/pull/15

Reply:
Just in case: as I have applied the suggestion by @rhendric now «links to …» is no longer there